app/src/androidTest/java/com/example/android/testing/notes/notes/AddNoteTests.java

```diff
@@ -82,12 +91,12 @@ public void addNoteWithImage() throws Exception {
         onView(withText(R.string.take_picture)).perform(click());
 
         // Click on shutter button
-        UiObject shutterBtn = mDevice.findObject(new UiSelector().resourceIdMatches(".*shutter_button"));
+        UiObject shutterBtn = mDevice.findObject(new UiSelector().text("Shutter"));
         shutterBtn.waitForExists(DEFAULT_TIMEOUT_MILLIS);
         shutterBtn.click();
 
         // Click on done button
-        UiObject doneBtn = mDevice.findObject(new UiSelector().resourceIdMatches(".*btn_done"));
+        UiObject doneBtn = mDevice.findObject(new UiSelector().resourceIdMatches(".*okay"));
```

Comment:
This fixes the issue with certain Samsung devices, but breaks others.
Please revert.